packages/common/src/styles/theme.ts

```diff
@@ -103,14 +122,16 @@ const themeOptions = {
     },
   },
   palette: {
-    darkGrey: '#555770',
+    darkGrey: { main: '#555770' },
```

Comment:
Because I did this, I then had to also go and update all the references to this colour